.gitignore

```diff
@@ -0,0 +1,2 @@
+yarn.lock
```

Comment:
I think it is actually recommended for yarn.lock to be committed, since multiple people developing the project would all be on the same locked version.